site/docs/relations/logical_relations.md

```diff
@@ -42,7 +42,15 @@ Read definition types are built by the community and added to the specification.
 | --------------- | ------------------------------------------------------------ | -------- |
 | Items           | An array Items (path or path glob) associated with the read  | Required |
 | Format per item | Enumeration of available formats. Only current option is PARQUET. | Required |
+| Slicing parameters per item | Information to use when reading a slice of a file. | Optional |
 
+##### Slicing Files
+
+A read operation may only read part of a file.  This is convenient, for example, when distributing
+a read operation across several nodes.  The slicing parameters are specified as byte offsets
+into the file.  Most file formats consist of indivisible "chunks" of data (e.g. parquet row groups).
+If this happens the chunk should only be read if the midpoint of the chunk (dividing by 2 and rounding
```

Comment:
Thank you for the explanations. I think actually whether to read this chunk depends on the internal logics, and maybe Spark's midpoint-logic is just one of them.

Reply:
This documentation should be in the proto otherwise it will out of date as soon as the proto is changed again.

Reply:
@cpcloud I can do that but it's still not clear to me what goes where.  I opened #111 for discussion

@rui-mo Thanks, I updated the wording so that it is up to the consumer which algorithm to use.

Reply:
@cpcloud I added some information to the proto file.